packages/zeebe-element-templates-json-schema/src/error-messages.json

```diff
@@ -328,6 +395,48 @@
       "properties",
       "properties"
     ],
+    "errorMessage": "Missing binding with `property`=`versionTag` as binding with `property`=`bindingType` and `value`=`versionTag` is set"
+  },
+  {
+    "path": [
+      "definitions",
+      "template",
+      "allOf",
+      1,
+      "allOf",
+      9,
+      "then",
+      "properties",
+      "properties"
+    ],
+    "errorMessage": "`property`=`bindingType` is not supported when using `property`=`externalReference`. Use `formId` with `bindingType`"
+  },
+  {
+    "path": [
+      "definitions",
+      "template",
+      "allOf",
+      1,
+      "allOf",
+      10,
+      "then",
+      "properties",
+      "properties"
+    ],
+    "errorMessage": "Binding with `property`=`processId` and `type`=`zeebe:calledElement` is required, when using a binding with `type`=`zeebe:calledElement`"
```

Comment:
Binding with property=processId is always required when you template a call activity.

Reply:
Not sure what you would like it changed to.
I don't think we should mention call activity but rather stick with calledElement